services-core/src/gas_price.rs

```diff
@@ -27,10 +28,24 @@ pub trait GasPriceEstimating: Send + Sync {
 pub async fn create_estimator(
     http_factory: &HttpFactory,
     web3: &Web3,
-) -> Result<Arc<dyn GasPriceEstimating + Send + Sync>> {
+) -> Result<Arc<dyn GasPriceEstimating>> {
     let network_id = web3.net().version().await?;
-    Ok(match gnosis_safe::api_url_from_network_id(&network_id) {
-        Some(url) => Arc::new(gnosis_safe::GnosisSafeGasStation::new(http_factory, url)?),
-        None => Arc::new(web3.clone()),
-    })
+    let mut estimators = Vec::<Box<dyn GasPriceEstimating>>::new();
+
+    if network_id == "1" {
```

Comment:
nit: maybe use a `is_mainnet()` helper for this logic (might not be needed right now but in general the network could be worth having as an enum available in the main function to more than just this dependency).